Marlin/Configuration.h

```diff
@@ -387,6 +387,10 @@ const bool Z_MAX_ENDSTOP_INVERTING = false; // set to true to invert the logic o
 // #define MANUAL_BED_LEVELING  // Add display menu option for bed leveling
 // #define MESH_BED_LEVELING    // Enable mesh bed leveling
 
+#if defined(MANUAL_BED_LEVELING)
+  #define MBL_Z_STEP 0.025
+#endif  // MANUAL_BED_LEVELING
+
```

Comment:
Add these lines to all the other `Configuration.h` files and this can be merged. It's a clean PR and a feature that doesn't interact with others, so I have no worries about new bugs with this.